_data/versions.yml

```diff
@@ -1350,30 +1350,44 @@ v3.0:
+    
+    - The `get`, `apply`, `create`, `delete`, and `replace` commands of `calicoctl` now include an optional `--namespace=<NS>` flag. Refer to the `calicoctl` [Command reference](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/) section for more details.
+    
+    - The `get` command of `calicoctl` now includes an optional `--all-namespaces` flag. Refer to the [calicoctl get](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/get) section for more information.
+    
+    - `calicoctl` no longer accepts the following flags in `get` commands: `--node=<NODE>`, `--orchestrator=<ORCH>`, `--workload=<WORKLOAD>`, and `--scope=<SCOPE>`. These options are now a part of the individual resources.
+    
+    - `calicoctl` no longer includes a `config` command. To achieve the equivalent functionality: refer to [Modifying low-level component configurations](https://docs.projectcalico.org/v3.0/reference/calicoctl/commands/#modifying-low-level-component-configurations).
 
     - You can now name [host](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/hostendpoint#endpointport) and [workload](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/workloadendpoint#endpointport) endpoint ports and reference them by name in your [policy rules](https://docs.projectcalico.org/v3.0/reference/calicoctl/resources/networkpolicy#ports).
 
     - The new `ApplyOnForward` flag allows you to specify if a host endpoint policy should apply to forwarded traffic or not. Forwarded traffic includes traffic forwarded between host endpoints and traffic forwarded between a host endpoint and a workload endpoint on the same host. Refer to [Using Calico to secure host interfaces](https://docs.projectcalico.org/v3.0/getting-started/bare-metal/bare-metal) for more details.
 
     - Calico now works with Kubernetes network services proxy with IPVS/LVS. Calico enforces network policies with kube-proxy running in ipvs mode for Kubernetes clusters. Currently only workload ingress policy is supported.
+    
 
-    **Limitations**
+
+    #### Limitations
 
     - **No upgrades**: Calico v3.0.0 ends support for etcd version 2. Existing customers must migrate their data to etcd version 3. The alpha release does not provide migration capabilities, nor does it support upgrades. We plan to add migration and upgrade support in the GA release.
 
-    - **Integrates only with Kubernetes and host endpoints**: the Docker, OpenStack, OpenShift, Mesos, DC/OS, and rkt orchestrators have not been tested and are not supported. (Calico v3.0.0 still supports Docker and rkt containers.)
+    - **Integrates only with Kubernetes and host endpoints**: the Docker, OpenStack, OpenShift, Mesos, DC/OS, and rkt orchestrators have not been tested and are not supported. (Calico v3.0.0 still supports Docker and rkt containers.) We plan to resume support for the Docker, OpenStack, OpenShift, Mesos, and DC/OS orchestrators in a future release.
```

Comment:
maybe we should say `Docker libnetwork` to avoid confusion between Docker the runtime (which is supported in this release) and Docker orchestration part (which uses our libnetwork plugin).
WDYT?

Reply:
I know that internally we think of libnetwork a lot but I am not sure it resonates quite as much externally. I think "orchestrator" is maybe a little more generally understood.